.github/workflows/lint.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             go.sum
       - uses: golangci/golangci-lint-action@v3
         with:
-          version: latest
+          version: v1.52.0
```

Comment:
Could we please avoid pinning golangci-lint to a specific version?

I've submitted #922 to fix the lints on the `v0.38.x` branch.

Reply:
Ok, will revert my commit and wait for #922 to be merged